src/test/rbd_mirror/test_ImageReplayer.cc

```diff
@@ -581,7 +581,7 @@ typedef ::testing::Types<TestImageReplayerParams<
                            cls::rbd::MIRROR_IMAGE_MODE_SNAPSHOT>>
     TestImageReplayerTypes;
 
-TYPED_TEST_CASE(TestImageReplayer, TestImageReplayerTypes);
+TYPED_TEST_SUITE(TestImageReplayer, TestImageReplayerTypes);
```

Comment:
Why not change it in `src/test/librbd/test_librbd.cc` as well? Did this even throw an error or are you just changing it because Googletest changed it?

Reply:
it's not a runtime error. i changed this because i spotted warnings when compiling it.

Reply:
Thanks -- I guess it doesn't warn under GCC.